test/plots/render.ts

```diff
@@ -0,0 +1,37 @@
+import * as Plot from "@observablehq/plot";
+import * as d3 from "d3";
+import * as htl from "htl";
+
+export async function renderInstantiatedScales() {
```

Comment:
The file name “render.ts” isn’t very descriptive.

And I think this would be better as a unit test instead of a snapshot test since most of it is about checking the exposed scales, not the visual appearance of the chart.